otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -97,9 +97,10 @@ replaces:
 
   # ----------------------------------------------------------------------------
   # Customized log collection
-  # ToDo: Remove after using v0.19.0 or newer in upstream.
-  # Replacing due to https://github.com/open-telemetry/opentelemetry-log-collection/pull/189
-  - github.com/open-telemetry/opentelemetry-log-collection => github.com/open-telemetry/opentelemetry-log-collection v0.19.0
+  # ToDo: Remove after listed PRs are in upstream.
+  # - https://github.com/open-telemetry/opentelemetry-log-collection/pull/189
+  # - https://github.com/open-telemetry/opentelemetry-log-collection/pull/212
+  - github.com/open-telemetry/opentelemetry-log-collection => v0.18.1-0.20210706124509-fd4dbb8a26b6
```

Comment:
Why does this say `0.18.X`? 

EDIT: doesn't matter I've just now noticed that there is no 0.19.0 in `github.com/open-telemetry/opentelemetry-log-collection` (how come? 🤷 )